src/components/form/Input/Input.js

```diff
@@ -0,0 +1,29 @@
+import { StyledInput } from "./Input.styled";
+
+export const Input = ({
+  type,
+  label,
+  field,
+  value,
+  handleChange,
+  errors,
+  isTouched,
+  handleBlur,
+}) => {
+  return (
+    <StyledInput>
+      <div>
+        <label htmlFor={field}>{label}</label>
+        <input
+          type={type}
+          id={field}
+          name={field}
+          value={value}
+          onChange={handleChange}
+          onBlur={() => handleBlur(field)}
+        />
+      </div>
+      {errors[field] && isTouched[field] === true && <p>{errors[field]}</p>}
```

Comment:
remove passing object of potential errors and just pass string message if error happens

```
        <Input
          type="text"
          label={"Description:"}
          ...
          errorMessage={formErrors["description"]}    <-
        />
  ```
  
  and use it that way
  
  ```
  </div>
  {errorMessage && errorMessage}
  </StyledInputt>
  ```